modules/integration/src/test/java/io/ballerina/messaging/broker/integration/standalone/jms/QueueLocalTransactionRollbackTest.java

```diff
@@ -0,0 +1,260 @@
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+ */
+
+package io.ballerina.messaging.broker.integration.standalone.jms;
+
+import io.ballerina.messaging.broker.integration.util.ClientHelper;
+import org.testng.Assert;
+import org.testng.annotations.Parameters;
+import org.testng.annotations.Test;
+
+import javax.jms.Connection;
+import javax.jms.ConnectionFactory;
+import javax.jms.Destination;
+import javax.jms.JMSException;
+import javax.jms.Message;
+import javax.jms.MessageConsumer;
+import javax.jms.MessageProducer;
+import javax.jms.Queue;
+import javax.jms.Session;
+import javax.naming.InitialContext;
+import javax.naming.NamingException;
+
+/**
+ * Test cases written to verify queue consumer and producer behavior in local transaction rollback operation
+ */
+public class QueueLocalTransactionRollbackTest {
+
+    @Parameters({"broker-port"})
```

Comment:
Have we used this parameter within the test? I could  not find a reference.

Reply:
If you look at each method parameter set as `String port`. Value of the parameter defined in `broker-auto-start-suite.xml`. TestNG runtime injects value when running the test case.